File: 프로그래머스/1/134240. 푸드 파이트 대회/푸드 파이트 대회.js
function solution(food) {
    var answer = '';
    let result = '';
    for(let i=1; i<food.length; i++) {
        if(food[i]/2 < 1) continue;
        else {
            for(let j=0; j<Math.floor(food[i]/2); j++) {
                result += String(i);
            }
        }
    }
    let reverse = result.split('').reverse().join('');
    answer += result + 0 + reverse;
    return answer;
}

File: 프로그래머스/1/134240. 푸드 파이트 대회/푸드 파이트 대회.java
public class Solution {
    public String solution(int[] food) {
        StringBuilder result = new StringBuilder();
        for (int i=0; i<food.length; i++) {
            if (food[i]/2 < 1) continue;
            else {
                for (int j=0; j<food[i]/2; j++) {
                    result.append(String.valueOf(i));
                }
            }
        }
        StringBuilder reverse = new StringBuilder(result);
        reverse.reverse();
        return result.toString() + "0" + reverse.toString();
    }
}